style.css

```diff
@@ -62,10 +69,56 @@
     justify-content: center;
     text-align: center;
     margin-top: 3rem;
+    
 }
 
 .header_image {
     width: 200px;
-    border: 1px solid black;
-    border-radius: 50%;
+    border-radius: 50px;
+    border: gray solid;
+}
+
+.container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.grid-wrapper {
+    display: grid;
+    grid-gap: 20px;
+    grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+    align-items: center;
+    justify-content: center;
+    margin: 0;   
 }
+
+.box-image {
+    margin: 10px;
+}
+
+.imago {    
+    border: .5px gray;
+    border-radius: 50px;
+    height: 300px;
+    width: 300px;
+    margin: 20px;
+}
+
+.ft {
+    background: gray;
+    display: flex;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 20px;
+    font-size: 15px;
+    color: black;
+}
+
+.img-portada {  
```

Comment:
Same, try to have more descriptives names.

Reply:
And ready